samples/istio/quickstart/experiment.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: iter8.tools/v2alpha2
+kind: Experiment
+metadata:
+  name: istio-quickstart
+spec:
+  # target identifies the service under experimentation using its fully qualified name
+  target: bookinfo-iter8/productpage
+  strategy:
+    # this experiment will perform an A/B test
+    testingPattern: A/B
+    # this experiment will progressively shift traffic to the winning version
+    deploymentPattern: Progressive
+    actions:
+      # when the experiment completes, promote the winning version using kubectl apply
+      finish:
+      - task: common/exec
+        with:
+          cmd: /bin/bash
+          args: 
+          - "-c"
+          - kubectl -n {{ .namespace }} apply -f {{ .promote }}
```

Comment:
Replacing {{ .namespace }} with the string bookinfo-iter8 will simplify line 21, eliminate the need for lines 42, 43, 57, and 58. I feel this simplicity is desirable.

Reply:
namespace is needed for the metrics we have defined; it will be necessary in the versionInfo anyway.

Reply:
Yes, agreed. Using `namespace` variable within metrics is a good idea. {{ .namespace }} placeholder in line  21 above doesn't serve any purpose and only makes the kubectl apply command needlessly cryptic, since  {{ .namespace }} is not dynamically interpolated variable.